experimental/generation/generator/packages/library/templates/generator.lg

```diff
@@ -74,275 +74,252 @@
 # importLG(name)
 -[${name}](${prefix}-${name}.${locale}.lg)
 
->> Template lists
-
-# genericPropertyTemplates()
-- ${[`${type}Property.lg`, `generic-missing.dialog`, `generic-help.dialog`, `generic-clear.dialog`, `generic-show.dialog`, `generic-change.dialog`]}
-
-# propertyTemplates()
-- ${[`${type}Property.lg`, `${type}-missing.dialog`, `${type}-help.dialog`, `${type}-clear.dialog`, `${type}-show.dialog`, `${type}-change.dialog`]}
+>> Enity lists
+# unitsEntities(entity)
+- ${union([entity], if(propertySchema.$units, ['number'], []))}
 
-# genericEntityTemplates()
-- ${[`${entity}Entity.lu`, `${entity}Entity.lg`, `genericEntity-add.dialog`, `genericEntity-remove.dialog`]}
-
-# entityTemplates()
-- ${[`${entity}Entity.lu`, `${entity}Entity.lg`, `${entity}Entity-add-${type}.dialog`, `${entity}Entity-remove-${type}.dialog`]}
+>> Template lists
 
-# propertyLGTemplates(type)
-- ${`${locale}/${type}Property.${locale}.lg`}
+# propertyLGTemplates(template)
+- ${`${locale}/${template}Property.${locale}.lg`}
+
+# standardTemplates()
+- ${[
+        "genericEntity.lu",
+        "genericEntity.lg",
+        "genericProperty.lg",
+        "genericEntity-add.dialog",
+        "genericEntity-remove.dialog",
+        "generic-missing.dialog",
+        "generic-clear.dialog",
+        "generic-show.dialog",
+        "generic-change.dialog",
+        "generic-help.dialog"
+    ]}
+
+# dateFormatTemplates()
+- ${[
+        "genericEntity.lu",
+        "genericEntity.lg",
+        "genericProperty.lg",
+        "dateFormat-add.dialog",
```

Comment:
Could we incorporate dateFormat-add.dialog into genericEntity-add.dialog as the specific case?

Reply:
I'm going to leave this for now.

---
In reply to: [570805648](https://github.com/microsoft/BotBuilder-Samples/pull/3061#discussion_r570805648) [](ancestors = 570805648)